content/blog/newsletter_2022-02-05.de.md

```diff
@@ -0,0 +1,235 @@
+---
+title: XMPP Newsletter Dezember 2021 und Januar 2022
+author: Anoxinon e.V. 
+date: 2022-02-05
+categories: ['Newsletter']
+---
+
+*Info: [Anoxinon e.V.](https://anoxinon.de/) publiziert die deutsche Übersetzung des unter der CC by-sa 4.0 Lizenz stehenden XMPP Newsletters für die XSF Foundation. Den [Originalartikel findest Du im Blog der XSF](https://xmpp.org/2022/02/the-xmpp-newsletter-december-2021-january-2022/). Übersetzung und Korrektur der dt. Version von Anoxinon e.V..*
```

Comment:
I think this is not needed in the official hosting

Reply:
Done